.stylelintignore

```diff
@@ -1,5 +1,8 @@
 node_modules/
 demo/public/
+demo/node_modules/
```

Comment:
Same here. Just `node_modules` and `__sapper__`. I don't think there is ever a case where we might need to include one of them